src/scss/form.scss

```diff
@@ -132,6 +132,7 @@
          margin-right: $spacing-x-small;
 
          &__label {
+            margin-bottom: 0;
```

Comment:
I think we should have a variable for "no margin". Otherwise it will be hard to customize this.